code/src/App.js

```diff
@@ -1,9 +1,93 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import { LikeCounter } from "LikeCounter";
+import { Loader } from "Loader";
+import { Thought } from "Thought";
+import { Form } from "./Form"
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+  const [messageOK, setMessageOK] = useState(true);
+  const [loading, setLoading] = useState(true);
+  const initialLikeCount = () => Number(window.localStorage.getItem("count")) || 0
+  const [likeCount, increaseLikeCount] = useState(initialLikeCount);
+
+  useEffect(() => {
+    getThoughts();
+  }, []);
+
+  useEffect(()=>{
+    window.localStorage.setItem("count",likeCount);
+  },[likeCount])
+
+  const getThoughts = () =>{
+    fetch("https://happy-thoughts-technigo.herokuapp.com/thoughts")
+    .then((res) => res.json())
+    .then((data) => {
+      setThoughts(data);
+      setLoading(false);
+    });
+  }
+
+  const handleLikes = () => {
+    increaseLikeCount((likeCount) => likeCount+1);
+  }
+
+  const handleSubmit = (newMessage) => {
+    postThought(newMessage);
+  }
+
+  const postThought = (message) => {
+    const myHeaders = new Headers();
+    myHeaders.append("Content-Type", "application/json");
+    const messageToPost = JSON.stringify({"message": message});
```

Comment:
left-side `message` shouldn't be wrapped inside quotes, it's supposed to be key of the object, and they are not strings or anything else, just names. Needs to be fixed